db/seedData/generatePGdata.js

```diff
@@ -0,0 +1,106 @@
+const faker = require('faker');
+
+const placeTypes = ['bakery', 'bar', 'cafe', 'restaurant', 'meal_delivery', 'meal_takeaway'];
+
+function generateRestaurantDataBatch(round) {
+  if (round % 100 === 0) {console.log(round);}
+  if (round > 1000) {
+    return Promise.resolve(null);
+  }
+  let data = [];
+  for (var x = 0; x < 10000; x++) {
+    let restId = (10000 * round) + x;
```

Comment:
Not essential, but you might consider pulling your batch size (looks like 10,000) into a variable so you can just reference it throughout the script or easily alter it.